management-app/src/main/java/com/nearsoft/fgaribay/mgmt/ManagementApplication.java

```diff
@@ -1,9 +1,14 @@
 package com.nearsoft.fgaribay.mgmt;
 
+import com.nearsoft.fgaribay.mgmt.config.BrokerConfig;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
 
 @SpringBootApplication
+@PropertySource("classpath:common.properties")
```

Comment:
See the comment in `common/src/main/resources/common.properties` file and try using a Spring profile for this common settings.